localization/languages/hu.json

```diff
@@ -6,28 +6,28 @@
         these are the items displayed in the menu when you right-click on a page
         */
     "addToDictionary": null, //missing translation
-    "pictureInPicture": null, //missing translation
+    "pictureInPicture": "Kép a képben", //missing translation
```

Comment:
Can you remove the `//missing translation` comments where you've added translations?

Reply:
Of course.